cfgov/v1/models/molecules.py

```diff
@@ -78,7 +78,7 @@ class FormFieldWithButton(blocks.StructBlock):
     btn_text = blocks.CharBlock(max_length=100, required=False)
 
     required = blocks.BooleanBlock(required=False)
-    id = blocks.CharBlock(max_length=100, required=False)
+    id = blocks.CharBlock(max_length=100, required=False, help_text="Type of form i.e emailForm, submission-form")
```

Comment:
Probably fine for now. Maybe say it should be unique though (I'm assuming it needs to be?).